Automattic/themes
# Fix/varia children pullquote color issue

#### Changes proposed in this Pull Request:

Changed the inheritance of the pullquote block background color so that the selected color from the editor is more specific.

#### How to test 

Create a new Pullquote block, make it solid color, change the main color and see if the frontend has the same color as the editor.

#### Screenshots

Varia: 

![](https://d.pr/i/PI5Sts+)

Stow: (Has a color annotation issue still in progress so one of the colors is not showing correctly due to that)

![](https://d.pr/i/sBaR6b+)


Morden: 

![](https://d.pr/i/viGNnX+)


#### Related issue(s):
#2485
